rotkehlchen/chain/ethereum/modules/eth2/beacon.py

```diff
@@ -246,7 +247,7 @@ def get_validator_data(
                 if queried_beaconchain and (exit_epoch := entry.get('exitepoch', 0)) != 0:
                     exited_ts = epoch_to_timestamp(exit_epoch)
                 else:  # query this index from beaconchain to see if exited
-                    indices_mapping_to_query_beaconchain[entry[index_key]] = idx
+                    indices_mapping_to_query_beaconchain[deserialize_int(entry[index_key])] = idx
```

Comment:
Yes I think this is required here too. Notice that in line 253 this is also done.

In line 264 it's not done (I assume by accident) but it's fine since it's beaconchain website that's queried only there